icu4c/source/test/testdata/BUILDRULES.py

```diff
@@ -169,6 +170,31 @@ def generate_conv(config, io, common_vars):
         )
     ]
 
+def generate_lstm(config, io, common_vars):
+    basenames = [
+        "Thai_graphclust_model4_heavy",
+        "Thai_codepoints_exclusive_model5_heavy",
+        "Burmese_graphclust_model5_heavy"
```

Comment:
Since all you're doing is converting some txt files to rb files, just add these new files to the list up above in generate_rb

Reply:
done